frontend/docs/pages/typescript-sdk/creating-a-workflow.mdx

```diff
@@ -0,0 +1,113 @@
+# Creating a Workflow
+
+To create a workflow, simply create a new `Workflow` object.
+For example, a simple 2-step workflow would look like:
+
+```ts
+import Hatchet from "@hatchet/sdk";
+import { Workflow } from "@hatchet/workflow";
+
+const hatchet = Hatchet.init();
+
+const workflow: Workflow = {
+  id: "example",
+  description: "test",
+  on: {
+    event: "user:create",
+  },
+  steps: [
+    {
+      name: "step1",
+      run: (input, ctx) => {
+        console.log("executed step1!");
+        return { step1: "step1" };
+      },
+    },
+    {
+      name: "step2",
+      parents: ["step1"]
+      run: (input, ctx) => {
+        console.log("executed step2!");
+        return { step2: "step2" };
+      },
+    },
+  ],
+};
+```
+
+You'll notice that the workflow defines a workflow trigger (in this case, `on_events`), and the workflow definition. The workflow definition includes a series of steps which is simply an array of `Step` objects. Each step has a `run` prop which is a function that takes an `input` and a `context`. The `context` argument is a `Context` object, which contains information about the workflow, such as the input data and the output data of previous steps.
+
+To create multi-step workflows, you can use `parents` to define the steps which the current step depends on. In the example, `step2` will not invoke until after `step1` completes.
+
+## Getting Access to the Input Data
+
+You can get access to the workflow's input data, such as the event data or other specified input data, by using the `context.workflow_input()` method on the `context`. For example, given the following event:
```

Comment:
```suggestion
You can get access to the workflow's input data, such as the event data or other specified input data, by using the `context.workflowInput()` method on the `context`. For example, given the following event:
```

As much as I don't like the inconsistency, I think it's important that we camelCase here and snake_case in python. It just feels more natural and will probably more closely align with most people's linters.